app/js/components/queryBuilder/queryParamSelector/_queryParamSelector.scss

```diff
@@ -61,7 +61,10 @@
 		box-shadow: 0 2px 5px rgba($black, 0.5);
 		cursor:pointer;
 	}
-
+	.disease-card:hover{
```

Comment:
instead of having to rewrite all that scss again, you could do:

```
.gene-card,
.disease-card {
    &:hover {
        box-shadow: ...
    }
}
```

Reply:
Aha! Much cleaner. I'll make that revision.